frontend/src/features/recommendation/SearchFilter.tsx

```diff
@@ -23,9 +24,15 @@ function SearchFilter() {
   };
 
   return (
-    <Box color="text.secondary">
+    <Box color="text.secondary" marginBottom={1}>
       <CollapseButton expanded={expanded} onClick={handleExpandClick} />
       <Collapse in={expanded} timeout="auto" unmountOnExit>
+        {filterParams.get('uploader') && (
+          <UploaderFilter
+            value={filterParams.get('uploader') ?? ''}
+            onDelete={() => setFilter('uploader', '')}
+          />
+        )}
```

Comment:
Sorry to bother you again about the margin. With the current branch, the padding of the other filters conflicts with the "delete" button, which is only partially clickable:
![image](https://user-images.githubusercontent.com/4726554/147778236-e1c11b99-6865-4201-9ddc-42670231b5e5.png)

I think that `marginButtom` should be applied around the `UploaderFilter` specifically:
```jsx
          <Box marginBottom={1}>
            <UploaderFilter
              value={searchParams.get('uploader') ?? ''}
              onDelete={() => setFilter('uploader', '')}
            />
          </Box>
```